apps/web/pages/index.tsx

```diff
@@ -35,17 +40,41 @@ const Home = () => {
             type="button"
             className="inline-flex items-center justify-center rounded-md border border-transparent bg-indigo-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2 sm:w-auto"
             onClick={() => {
-              setForm(!displayForm);
+              setForm(!displayForm); //this will change the true or false statement, thus determining if there will be a new info display
             }}
           >
-            Create Post
+            Create/Cancel Post
```

Comment:
[nit] I think the UX on this button can be a bit confusing if it says Create/Cancel, we could instead conditionally render what the button label is based on whether the form is visible or not:
```suggestion
            {`${displayForm ? "Cancel" : "Create"} Post`}
```